docs/projects/reviews/2024/2024-vc-api-implementation-annual.md

```diff
@@ -0,0 +1,79 @@
+
+## Goals
+
+### Performance Against Prior Goals
+
+The project proposal mentioned the following aims in the "Alignment with the OpenWallet Foundation Mission" section:
+- Advance the adoption of interoperable and privacy enhancing digital wallets.
+- Promote the development and proliferation of the VC API specification itself
+
+Progress towards the above objectives has been small over the past year.
+However, the main functional enhancement undertaken in recent months (the migration of core key & credential operations to Credo) sets up the project well for progress on the objectives in the year ahead.
+For example, listing in the VC Playground (https://vcplayground.org/) is an opportunity to further both of the above objectives, however this was blocked by that lack of flexibility with JSON-LD context document loading in the implementation prior to the Credo integration.
+
+### Next Year's Goals
+
+The current goals of the project are to:
+1. Increase use and adoption of the project, both by existing user organizations (Elia and Energy Web) and new organizations. The target metric is between 1 and 5 new organizations using the app in projects within the year.
+2. Support the development of the VC API specification.
+3. Provide feedback into the development of other OWF projects.
+
+On-going and planned work items which further these goals are detailed below.
+
+#### Migrating to Credo for key and credential operations
+
+GitHub Issue: https://github.com/openwallet-foundation-labs/vc-api/issues/20
+
+The benefits of this item include:
+- Synergy with an existing OWF project, allowing work on the VC API implementation to feed back into other OWF work
+  - Examples of this are the following GitHub issues:
+    - [Credo Issue #2038- Make verifiableCredential property on W3cPresentation optional](https://github.com/openwallet-foundation/credo-ts/issues/2038) 
+    - [Credo Issue #2043 - Fix W3cCredentialsApi verifyCredential types](https://github.com/openwallet-foundation/credo-ts/issues/2043)
+- The ability to use load additional JSON LD contexts, unlocking new use cases
+- A clearer path to supporting further VC technology such as SD-JWT and BBS signatures
+
+Work on this item can be seen in the following PR: https://github.com/openwallet-foundation-labs/vc-api/pull/36
+
+#### Improving documentation for technology adoption decision makers
+
+GitHub Issue: https://github.com/openwallet-foundation-labs/vc-api/issues/29 
+
+This item aims to improve the ability of technology selection decision makers, such as CTOs or Software Architects, to assess whether or not the VC API implementation is suitable for their use cases and technology context.
+The key output is a new documentation site.
+
+#### Listing in the VC Playground
+
+GitHub Issue: https://github.com/openwallet-foundation-labs/vc-api/issues/28
+
+The VC Playground is a demonstration application for Verifiable Credential issuance and verification: https://vcplayground.org.
```

Comment:
Who supports/runs the VC Playground?

Reply:
@tkuhrt I know that Digital Bazaar assists with running and developing the VC Playground. There may be other supporters that I am not aware of.